.github/workflows/pg-docker.yml

```diff
@@ -0,0 +1,52 @@
+name: Builds and publishes a docker image for the back end service
+
+on:
+  push:
+    paths:
+      - database/**
+      - .github/workflows/pg-docker.yml
+    branches: [main]
+    tags:
+      - "v*"
```

Comment:
what's the reason for running this on tags?

Reply:
No reason, i have removed it